[FR][Windows] Build Failed for Visual Studio 2026

As title.
```pwsh
# etc...
    C:\Program Files\Microsoft Visual Studio\18\Insiders\MSBuild\Microsoft\VC\v180\Microsoft.CppBuild.targets(473,5): error MSB8020: The build tools for Visual Studio 2019 (Platform Toolset = 'v142') cannot be found. To build using the v142 build tools, please install Visual Studio 2019 build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution". [D:\ProgramDataD\MiscLang\07.02-Rust\rarely\mupdf-rs\target\release\build\mupdf-sys-f1c2f651b8315074\out\build\platform\win32\libharfbuzz.vcxproj]

      0 Warning(s)
      7 Error(s)

  Time Elapsed 00:00:00.40

  --- stderr

  msbuild invocation failed with status 1
```

# Environment 
```pwsh
> fastfetch -l none -s "OS:Kernel:Shell:WM:Terminal:CPU:Memory:Swap"
OS: Windows 11 Pro Insider Preview (26H2) x86_64
Kernel: WIN32_NT 10.0.26300.8935
Shell: PowerShell 7.7.0
WM: Desktop Window Manager 10.0.26100.8935
Terminal: Windows Terminal Preview 1.25.1912.0
CPU: 12th Gen Intel(R) Core(TM) i7-12850HX (16+8) @ 4.80 GHz
Memory: 19.15 GiB / 31.69 GiB (60%)
Swap: 1.20 GiB / 10.50 GiB (11%)
> rustc --version
rustc 1.97.0 (2d8144b78 2026-07-07)
```

--- a/mupdf-sys/msbuild.rs
+++ b/mupdf-sys/msbuild.rs
@@ -164,6 +164,7 @@ impl Msbuild {
 
         let platform_toolset = env::var("MUPDF_MSVC_PLATFORM_TOOLSET").unwrap_or_else(|_| {
             match find_vs_version() {
+                Ok(VsVers::Vs18) => "v145",
                 Ok(VsVers::Vs17) => "v143",
                 _ => "v142",
             }

